src/blog/2024-11-02/index.md

```diff
@@ -0,0 +1,107 @@
+summary: Release note for Neotokyo; Rebuild Pre-Alpha v9.0
+author: brysondev
+
+# Pre-Alpha v10.0 Release Note
+2024-11-02 brysondev
+
+* [Download v10.0-prealpha build](https://github.com/NeotokyoRebuild/neo/releases/tag/v10.0-prealpha)
+* [Install NT;RE (client)](/guide/install/)
+* [GitHub Issues (Bug reports and feature requests)](https://github.com/NeotokyoRebuild/neo/issues)
+
+## New features
+
+### Deathmatch
+
+Deathmatch is the same as TDM with both CTG capture points & ghosts disabled, and enables respawning. 
+
+The new ConVars for these settings are as follows:
+
+* `neo_vote_game_mode`- 1 by default, Vote on game mode to play. TDM=0, CTG=1, VIP=2, **DM=3**
+* `neo_dm_round_timelimit` - 10.25 by default, DM round time limit, in minutes.
+* `neo_sv_dm_win_xp` - 50 by default, XP limit to win the match.
+* `neo_sv_dmspawn_useent` - Default 0, override to use map's deathmatch spawn entity over the dmspawn loc file.
+* `neo_sv_dm_max_class_dur` - Default 5s, maximum duration you're allowed to pick a class post-spawning in.
+* `neo_sv_warmup_godmode` - 0 by default, if 1, everyone takes no damage at all, regardless of bullets, fall, or grenades during idle or warmup state.
+
+In addition, new ConCommands (for local servers only) have been added to customize dm spawning locations:
+
+* `neo_sv_dmspawn_create` - Create a dmspawn spawn.
+* `neo_sv_dmspawn_removeall` - Remove all dmspawn spawns.
+* `neo_sv_dmspawn_printlocs` - Print all coordinates of the dmspawn spawns.
+* `neo_sv_dmspawn_save` - Save the map's loc dmspawn file.
+* `neo_sv_dmspawn_load` - Load the map's loc dmspawn file.
+* `neo_sv_dmspawn_teleportnext` - Teleport to the next dmspawn spawn.
+* `neo_sv_dmspawn_mapinfo` - Check amount of dmspawn spawns and if it can DM.
+
+If you wish to customize the spawns for a dedicated server, be sure to copy the generated dmspawn .loc files to the server.
+
+### Streamer Mode
+
+NT;RE is now able to detect OBS running and introduces safeguards for streamers. This includes no user avatars and replaces all names with preset generic names.
+
+Streamer mode can be enabled in the options menu, or by typing `neo_cl_streamermode 1` in the developer console. The auto OBS detection can also be enabled in the options menu or with `neo_cl_streamermode_autodetect_obs 1` (This requires a game restart to take effect.)
+
+### Clan Tags
+
+Clan Tags can now be set in the options menu, or by typing `neo_clantag <tag>` in the developer console. Server owners can opt out of clan tags being show by adding `neo_sv_clantag_allow 0` to their server configuration files. If all players on a team have a matching clan tag, the team's name will reflect the tag set.
+
+### Viewmodel Only Lean
+
+Players now have the option to enable viewmodel only leaning. This prevents a roll effect to the camera view while leaning. Option can be changed in the options menu, or by typing `cl_neo_lean_viewmodel_only 1` in the developer console.
+
+## General fixes/changes
```

Comment:
Can you split the fixes and new addition, this bullet point section is quite long.